examples/from-file/names.csv

```diff
@@ -0,0 +1,280 @@
+SMITH
+BROWN
+WILSON
+STEWART
+ROBERTSON
```

Comment:
can you trim this down to be only a few lines

Reply:
How many?

Reply:
like 5?

maybe don't call it names, so it doesn't confuse with the ofType names